inst/templates/tidy-eval.R

```diff
@@ -16,6 +16,7 @@
 #' @export           sym syms ensym
 #' @importFrom rlang expr exprs enexpr
 #' @export           expr exprs enexpr
+#' @importFrom rlang UQ UQS !! !!!
```

Comment:
There's no need to import `!!` and `!!!` since those functions are never actually called

Reply:
Thanks for the explanation! Will amend accordingly.